moonshine/demo/README.md

```diff
@@ -182,9 +153,10 @@ This is an example of the Moonshine base model being used to generate live capti
 (env_moonshine_demo) parallels@ubuntu-linux-2404:~$
 ```
 
-For comparison this is the Faster-Whisper int8 base model on the same instance.
+For comparison, this is the `faster-whisper` int8 base model on the same instance.
```

Comment:
nit: the example is float32 precision (not int8): this is a copy of my error.  I can fix this in the parent branch.